README.md

```diff
@@ -80,6 +80,13 @@ python main.py test --exp_name laneatt_r34_tusimple
 ```
 The results on TuSimple and LLAMAS should match exactly the ones reported in the paper. The results on CULane will deviate in the order of 0.1% (as shown in the CULane table below), since the metric reported on the paper was computed with the official code (C++), while this script will compute it using our implementation (which is much faster and in Python). The official metric implementation is available [here](https://github.com/XingangPan/SCNN/tree/master/tools/lane_evaluation).
 
+3. Run demo for image or video:
```

Comment:
Please place those instructions on a separate subsection. The demo is not a step necessary to reproduce the paper's results. Also, keep the commands according to Linux's standards (e.g., paths are divided by a forward slash, CLI options start with -- unless it is the shorter version).